src/scenes/PlayerScene.js

```diff
@@ -0,0 +1,99 @@
+import Phaser from 'phaser';
+
+import Player from '../sprites/Player';
+
+import { GUI } from 'dat.gui/build/dat.gui.js';
+
+export default class PlayerScene extends Phaser.Scene {
+
+  constructor() {
+    super( { key: 'PlayerScene' });
+
+    this.in_game = false;
+    this.player = null;
+    this.socket = window.socket;
+    this.other_players = new Map();
+    if (__DEV__ === 'true') {
+      this.dat_gui = new GUI();
+    }
+  }
+
+  createWorld() {
+    this.grid = this.add.image(0, 0, 'grid');
+    this.grid.x = this.grid.width / 2;
+    this.grid.y = this.grid.height / 2;
+    this.cameras.main.setBounds(0, 0, this.grid.width, this.grid.height);
+    this.cameras.main.startFollow(this.player);
+  }
+
+  createPlayer() {
+    console.log('createPlayer');
+    let player_size = 20;
+    this.player = new Player({
+      scene: this,
+      key: 'player',
+      x: player_size + Math.floor(Math.random() * (this.grid.width - player_size * 2)),
+      y: player_size + Math.floor(Math.random() * (this.grid.height - player_size * 2)),
+      map_bounds: {
+        tl_x: 0 + player_size,
+        tl_y: 0 + player_size,
+        br_x: this.grid.width - player_size,
+        br_y: this.grid.height - player_size
+      },
+      controlled: true
+    });
+    if (__DEV__ === 'true') {
+      let guiPlayer = this.dat_gui.addFolder('Player');
+      guiPlayer.add(this.player, 'x', 0, this.grid.width).listen();
+      guiPlayer.add(this.player, 'y', 0, this.grid.height).listen();
+      guiPlayer.open();
+    }
+    this.in_game = true;
+    this.socket.emit('newPlayer', {x: this.player.x, y: this.player.y});
+  }
+
+  createRemotePlayer(data) {
+    console.log('createRemotePlayer');
+    let player_size = 20;
```

Comment:
I think we should move this into the constructor, because we do not change this value and as far as I see this is exactly the same as in the `createPlayer` function.